js/templates/modals/editListing/editListing.html

```diff
@@ -150,7 +156,7 @@ <h2 class="h4 clrT"><%= ob.polyT('editListing.sectionNames.photos') %></h2>
       </section>
 
       <section class="js-scrollToSection js-sectionShipping shippingSection <% if (ob.listing.metadata.contractType !== 'PHYSICAL_GOOD') print('disabled') %> <% if (ob.listing.shippingOptions.length) print('hasShippingOptions') %>">
-        <div class="pad overflowAuto emptyShippingOptionsSection contentBox padMd clrP clrBr clrSh3 tx3">
+<!--         <div class="pad overflowAuto emptyShippingOptionsSection contentBox padMd clrP clrBr clrSh3 tx3">
```

Comment:
Could you please log an issue for that? There's a good amount of JS that deals with that guy which I would want to clean up.